tiledb/sm/subarray/subarray.cc

```diff
@@ -1666,8 +1678,9 @@ uint64_t Subarray::range_idx(const std::vector<uint64_t>& range_coords) const {
 }
 
 uint64_t Subarray::range_num() const {
-  if (range_subset_.empty())
+  if (range_subset_.empty() && label_range_subset_.empty()) {
```

Comment:
```suggestion
  if (range_subset_.empty()) {
```
This function should fully ignore dimension labels.